docs/getting-started/android/release-deployment.md

```diff
@@ -1,10 +1,66 @@
-# Deploy to Google Play using fastlane
+# Deploy to Google Play using _fastlane_
 
-To be written soon, this will contain
+# Building your app
 
-- Building your app using _gradle_
-- Reference the screenshots docs
-- Deploy to Google Play for production
-    - Mention how to deploy to the beta track
-- Best practises around the deployment
-    - Order of commands
+_fastlane_ takes care of building your app by delegating to your existing Gradle build. Just add the following to your `Fastfile`:
+
+```ruby
+lane :playstore
+  gradle(
+    task: 'assemble',
+    flavor: 'Paid',
```

Comment:
Not sure if `flavor: 'Paid'` is helpful here. It won't work for most people if they copy and paste this, where assemble, Release would